blocks/links-bar-block/_index.scss

```
@@ -0,0 +1,21 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-links-bar {
+	&__menu {
+		a {
+			@include scss.block-components("links-bar__menu-link");
+			@include scss.block-properties("links-bar__menu-link");
```

```suggestion
			@include scss.block-components("links-bar-menu-link");
			@include scss.block-properties("links-bar-menu-link");
```

Like I said in the previous comment, we want the scss classes to be bem naming. And we want the tokens to be separated by dashes, like other tokens. Can you apply this change to other tokens only

---

updated